craft_application/services/lifecycle.py

```diff
@@ -153,6 +155,11 @@ def _init_lifecycle_manager(self) -> LifecycleManager:
         """
         emit.debug(f"Initialising lifecycle manager in {self._work_dir}")
         emit.trace(f"Lifecycle: {repr(self)}")
+        feature_args: dict[str, Any] = {}
+
+        if self._app.features.package_repository:
+            feature_args["package_repositories"] = self._project.package_repositories
```

Comment:
out of curiosity, why use a separate dict here?

Reply:
Because I am not sure if I should put it into `self._manager_kwargs`